### DISCUSSION TITLE
Add Belarusian and Ukrainian localizations

### DISCUSSION THREAD

**Discussion by Prominence** [paperless-ngx/paperless-ngx#558] [Feature Requests] [answered]
Hi there!
I''ll duplicate the subject, but is it possible to add support of Belarusian and Ukrainian localizations in the project?

**Comment by shamoon** [comment]
Hi @Prominence , just to clarify, are you offering to translate them or just asking for them to be available in the app? They would need to be translated first

**Reply by Prominence** [reply]
I'm asking to make them available on **crowdin** at least. Then, I could try my best to translate what I can. I've just finished Russian messages that weren't translated yet.
And I guess there will be a fallback translation in case of missing one on the selected language? 

To clarify, I'm from Belarus and could provide mostly Belarusian translations. But I'm sure that Ukrainian community is more larger than ours and there will be any people who could do that.

**Reply by shamoon** [reply]
Ah, great, thank you for translating! 

I added Belarusian for you, perhaps we should leave Ukrainian until someone is ready to translate that one?

**Reply by Prominence** [reply]
Great. Yes, add, please, just Belarusian for now then.

**Reply by tim-vogel** [reply]
> [...] perhaps we should leave Ukrainian until someone is ready to translate that one?

I am in favor of including any language that has been requested in the project. Maybe this way it will be easier to find supporters to do the translation. This should not be detrimental to the project.

**Reply by shamoon** [reply]
Er, I agree except that without someone to actually translate it will just be left to languish, like [Greek](https://crowdin.com/project/paperless-ngx/el) and [Hebrew](https://crowdin.com/project/paperless-ngx/he) currently. If someone is ready to translate they reach out here or via crowdin (which of course is necessary for them to do so), seems like a low-enough barrier for someone who is looking to spend a bunch of time working on a translation, no?

**Reply by tim-vogel** [reply]
I agree with you there. No matter which path is chosen, there are advantages and disadvantages.

**Reply by juahan** [reply]
Now that we are talking about the languages, Could Finnish be added, I could then start translating it when ever I have some time.

**Reply by shamoon** [reply]
Done. Everyone should let me / us know when translating is done, languages need to be added to the code once thats done.

**Reply by Prominence** [reply]
> Done. Everyone should let me / us know when translating is done, languages need to be added to the code once thats done.

I've just finished my work on Belarusian localization. All problems(if any found) I'll fix later after using Paperless-ngx on my machine for some time.

**Reply by shamoon** [reply, marked as answer]
Thanks! See #588
